tools/rp_storage_tool/src/bucket_reader.rs

```diff
@@ -381,18 +381,29 @@ impl Anomalies {
     }
 
     pub fn merge(&mut self, other: Anomalies) {
```

Comment:
I thought I ran `cargo fmt` to format the code before I merged these changes in. Where they formatted incorrectly?

Reply:
I ran `cargo fmt` on the code and it re-formatted that code.